private/bufpkg/bufmodule/digest.go

```diff
@@ -75,15 +76,15 @@ func (d DigestType) String() string {
 //
 // This reverses DigestType.String().
 //
-// Returns an error of type *ParseError if thie string could not be parsed.
+// Returns an error of type *bufparse.ParseError if the string could not be parsed.
 func ParseDigestType(s string) (DigestType, error) {
```

Comment:
See comment in `bufplugin/digest.go`: we considered splitting digest into its own shared package. Looking at this, it does make sense, but maybe it should be done in a follow-up.